test/unit/common/AndroidLauncher.test.ts

```diff
@@ -32,7 +33,7 @@ describe('Android Launcher tests', () => {
         const launcher = new AndroidLauncher('Pixel XL');
         await launcher.launchPreview('helloWorld', '~', undefined, 'browser', undefined, '3333');
 
-        expect(launchUrlMock.calledWith('http://10.0.2.2:3333/lwc/preview/c/helloWorld', 5572));
+        expect(launchUrlMock.calledWith('http://10.0.2.2:3333/lwc/preview/c/helloWorld', 5572)).to.be.true;
```

Comment:
I didn't know that `expect(someBoolVar)` doesn't actually check whether `someBoolVar === true` Instead it just ensures that `someBoolVar` is not `undefined`. So updated `all` of out unit tests to explicitly do the check.